tools/local-environment/package.json

```diff
@@ -28,7 +28,7 @@
     "@nomad-xyz/contracts": "file:/../../solidity/nomad-core",
     "@nomad-xyz/deploy": "file:/../../typescript/nomad-deploy/dist",
     "@nomad-xyz/sdk": "file:/../../typescript/nomad-sdk/dist",
-    "@nomad-xyz/test": "file:/../../typescript/nomad-tests/dist",
+    "@nomad-xyz/test": "file:/../../typescript/nomad-tests",
```

Comment:
does this still fail if you build tests?

Reply:
compiling tests fails with ```node_modules/hardhat/src/internal/hardhat-network/stack-traces/model.ts(278,55): error TS7030: Not all code paths return a value.```